stdpopsim/catalog/OrySat/demographic_models.py

```diff
@@ -0,0 +1,119 @@
+import msprime
+import stdpopsim
+
+_species = stdpopsim.get_species("OrySat")
+
+# population definitions that are reused.
+_rufipogon = stdpopsim.Population(id="RUF", description="Oryza rufipogon (wild rice)")
+_indica = stdpopsim.Population(id="IND", description="Oryza sativa indica")
+_tropical_japonica = stdpopsim.Population(
+    id="TRP", description="Tropical Oryza sativa japonica"
```

Comment:
I would use "TRJ" for the tropical japonica id.

Reply:
Okay